spec/controllers/base_indexer/items_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+describe BaseIndexer::ItemsController, type: :controller do
+  let(:my_instance) { instance_double(BaseIndexer::MainIndexerEngine) }
+  before do
+    allow(BaseIndexer::MainIndexerEngine).to receive(:new).and_return(my_instance)
+  end
+  describe 'PATCH/PUT update' do
+    it 'creates an indexing job' do
+      expect(my_instance).to receive(:index).with('bb1111cc2222', 'SEARCHWORKS' => true)
+      patch :update, druid: 'druid:bb1111cc2222', subtarget: 'SEARCHWORKS', use_route: :base_indexer
+      expect(response.status).to eq 200
+    end
+    it 'when something bad happens return a 202' do
+      expect(my_instance).to receive(:index).with('bb1111cc2222', 'SEARCHWORKS' => true).and_raise(StandardError)
+      patch :update, druid: 'druid:bb1111cc2222', subtarget: 'SEARCHWORKS', use_route: :base_indexer
+      expect(response.status).to eq 202
```

Comment:
Yep this is part of question outlined in #33. This is current behavior and hopefully we can do better. It will also return a `202` if for instance it only can delete one many subtargets.